wordpress-mobile/WordPress-iOS
# Reader Cards: Update Save for Later button title / style

Fixes #9424.

This PR reuses the changes from #9425, but for Reader cards instead of Reader detail views. It updates the colors / highlighted states for the Save button on Reader cards, as well as the title (which is visible in a regular width environment – landscape iPhone, or larger views on iPad):

![reader-card-save](https://user-images.githubusercontent.com/4780/40863371-78f1ffa4-65e7-11e8-9903-0c48b610b5b2.png)

**To test:**

* Build and run
* Navigate to a Reader stream that shows the Save button – Follow Sites should work. Note that the Discover feed doesn't show save buttons currently.
* Check that the appearance for the save button on the cards in the list view match those shown above.
